pkg/dynamicinstrumentation/diconfig/binary_inspection.go

```diff
@@ -73,6 +73,10 @@ func AnalyzeBinary(procInfo *ditypes.ProcessInfo) error {
 	if err != nil {
 		return fmt.Errorf("could not determine locations of variables from debug information %w", err)
 	}
+	stringPtrIdentifier := bininspect.FieldIdentifier{StructName: "string", FieldName: "str"}
+	stringLenIdentifier := bininspect.FieldIdentifier{StructName: "string", FieldName: "len"}
+	r.StructOffsets[stringPtrIdentifier] = 0
+	r.StructOffsets[stringLenIdentifier] = 8
```

Comment:
Why do we need to set these, won't `InspectWithDWARF` get them?

Reply:
I need to investigate further, but InspectWithDWARF was not getting them.